__tests__/endpoints.test.js

```diff
@@ -175,6 +175,15 @@ describe("/api/articles/:article_id/comments", () => {
         .expect(400)
         .then(({body}) => {
             expect(body.msg).toBe("Bad Request")
+        });
+    });
+    test("404: Not Found, when requesting comments on article with no comments", () => {
+        return request(app)
+        .get("/api/articles/2/comments")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("No Comments For This Article")
         })
```

Comment:
This test actually shouldnt fail, its a valid response to receive an empty array for when an article has no comments.